frontend/tsconfig.json

```diff
@@ -8,6 +8,8 @@
     "outDir": "build",
     "resolveJsonModule": true,
     "skipLibCheck": true,
+    "module": "es2020",
+    "moduleResolution": "node",
```

Comment:
hmm, maybe it has to do with the fact that `module` was added as a config prop? Anyway, since the errors are resolved after you added the `moduleResolution` prop, i'm feeling good about this diff!